### PR TITLE
PR Template

### PR DESCRIPTION
**Pull request by Tgenz1213**
- **Title**: Add pull request template
- **Description**: This pull request adds a standardized pull request template to the repository. The template is designed to ensure consistency and completeness in future pull requests by providing a structured format for contributors to follow.
- **Related Issues**: #909
- **Type**:
  - [x] Documentation
  - [ ] Feature
  - [ ] Bug Fix
  - [ ] Refactor
  - [ ] Other:

## Implementation Details

- [x] Changes are focused and solve the stated problem
- [x] Code follows project style guides
- [x] Complex logic is documented
- [x] No unnecessary complexity introduced

## Testing

- [ ] Unit tests added/updated
- [ ] Integration tests added/updated
- [x] Manual testing completed
- [x] All tests passing

## Documentation & Quality

- [x] Project documentation updated
- [x] Code reviewed for clarity
- [ ] Breaking changes clearly marked
- [ ] Dependencies documented

## Deployment Impact

- [ ] Database migrations required? [No]
- [ ] Configuration changes needed? [No]
- [ ] Breaking changes? [No]

## Additional Notes

This template will help streamline the review process by ensuring all necessary information is provided upfront. It is a documentation change and does not affect the codebase functionality.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Pull Request
```
Comment (surapuramakhil) [inline .github/pull_request_template.md:1]: remove this line

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Pull Request
+
+## Overview
+
+- **Title**: [Descriptive title of the changes]
```
Comment (surapuramakhil) [inline .github/pull_request_template.md:5]: add description, after title

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Pull Request
+
+## Overview
```
Comment (surapuramakhil) [inline .github/pull_request_template.md:3]: remove this line

**Review by surapuramakhil — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (Tgenz1213) [reply .github/pull_request_template.md]: Fixed in [f21530d](https://github.com/AIHawk-FOSS/Auto_Jobs_Applier_AI_Agent/pull/910/commits/f21530dc055f8fa5c1f1d449c814654489bc150a)

File: .github/pull_request_template.md (hunk 3 above)
Comment (Tgenz1213) [reply .github/pull_request_template.md]: Fixed in [f21530d](https://github.com/AIHawk-FOSS/Auto_Jobs_Applier_AI_Agent/pull/910/commits/f21530dc055f8fa5c1f1d449c814654489bc150a)

File: .github/pull_request_template.md (hunk 2 above)
Comment (Tgenz1213) [reply .github/pull_request_template.md:1]: Updated. Please verify this is what you described.

**Review by surapuramakhil — APPROVED** [review]

**Review by surapuramakhil — APPROVED** [review]

**Pull request merged**